lib/galaxy_test/selenium/test_workflow_run.py

```diff
@@ -32,6 +32,52 @@
 class TestWorkflowRun(SeleniumTestCase, UsesHistoryItemAssertions, RunsWorkflows):
     ensure_registered = True
 
+    @selenium_test
+    @managed_history
+    def test_workflow_export_file_native(self):
```

Comment:
```suggestion
    def test_workflow_export_file_rocrate(self):
```
I guess you meant this for the name?